src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -608,6 +611,107 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
       }
     }
 
+  def queryEntities(workspaceName: WorkspaceName, entityType: String, query: EntityQuery): Future[PerRequestMessage] = {
+    object AttributeStringifier {
```

Comment:
meh... it seems specific to this implementation _right now_ and I just assumed we would put it in a better place if another use for it came up.